app/templates/credits.html

```diff
@@ -0,0 +1,129 @@
+{% extends "base.html" %}
+
+{% block head %}
+{{ super() }}
+    <link href="{{ url_for('static', path='/credits_style.css') }}" rel="stylesheet">
+{% endblock %}
+
+{% block content %}
+    <h1>Say hello to our developers:</h1>
+    <div class="gallery">
```

Comment:
Prefer having this data in some kind of a JSON, and loading it using a for loop

Reply:
fixed :)